python/ray/tune/execution/ray_trial_executor.py

```diff
@@ -235,43 +234,50 @@ def __init__(
         else:
             self._trial_cleanup = None
 
+        # For printing used resources
         self._resource_updater = _ResourceUpdater(refresh_period)
 
-        self._has_cleaned_up_pgs = False
+        # Actor re-use
         self._reuse_actors = reuse_actors
-        # The maxlen will be updated when `setup(max_pending_trials)` is called
-        self._cached_actor_pg = deque(maxlen=1)
-        self._pg_manager = _PlacementGroupManager(prefix=_get_tune_pg_prefix())
+        self._max_staged_actors = 1
```

Comment:
this is used for more than reusing actors right?

Reply:
That's right, it generally controls how much autoscaling we trigger (and how many configuration we generate - e.g. we support creating infinite trials)